src/test/java/edu/cornell/kfs/module/ar/document/validation/impl/CuTotalAmountBilledToDateExceedsAwardTotalSuspensionCategoryTest.java

```diff
@@ -0,0 +1,143 @@
+package edu.cornell.kfs.module.ar.document.validation.impl;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kuali.kfs.coreservice.framework.parameter.ParameterService;
+import org.kuali.kfs.krad.document.DocumentBase;
+import org.kuali.kfs.module.ar.ArConstants;
+import org.kuali.kfs.module.ar.businessobject.InvoiceGeneralDetail;
+import org.kuali.kfs.module.ar.document.ContractsGrantsInvoiceDocument;
+import org.kuali.kfs.module.cg.businessobject.Award;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import edu.cornell.kfs.module.ar.CuArConstants;
+import edu.cornell.kfs.module.cg.businessobject.AwardExtendedAttribute;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Award.class, ContractsGrantsInvoiceDocument.class})
```

Comment:
Does the Award class need to be altered by PowerMockito for this test?

Reply:
nope, it didn't, thanks for noticing